autoload/laravel/goto.vim

```diff
@@ -91,17 +91,11 @@ endfunction
 
 ""
 " @private
-" Capture language name at cursor using trans() method
+" Capture language name at cursor using trans(), trans_choice() and __() method
 function! laravel#goto#language() abort
-  return s:find_name('\<trans\%(_choice\)\?([''"]\([^''".]\+\)[^''"]*[''"][,)].*$')
+  return s:find_name('\<__\|trans\%(_choice\)\?([''"]\([^''".]\+\)[^''"]*[''"][,)].*$')
```

Comment:
You need another set of parentheses around the "or": `...(__|trans(_choice)?)...`

Try:

```vim
return s:find_name('\<\%\(__\|trans\%(_choice\)\?\)([''"]\([^''".]\+\)[^''"]*[''"][,)].*$')
```

Reply:
It works perfect! I am not into regular expressions so deeply, i would have never got this right so quicky. I rebased the Pull Request to have only one commit.